### DISCUSSION TITLE
servings scale in edit mode

### DISCUSSION THREAD

**Discussion by TWART016** [TandoorRecipes/recipes#1502] [General]
I a recipe `View Mode` I can change the servings and the Ingredients will scale.
In recipe `Edit Mode` I can change the servings and the Ingredients do not scale. 

Is there another way to change the ingredients value for a recipe instead of calculate manually? Cookidoo for example have a serving of 4 or more but normally I just need 2.

**Comment by vabene1111** [comment]
there is no "re scale" option in the editor yet. This makes sense as you tell the system the following list of ingredients creates the following number of servings. What might be useful would be a "change base servings" function where you could scale up or down the entire recipe in the editor to basically change the default servings of that recipe for the future.
